recipes/cds-crtools/build-python-awg.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+# build out-of-tree
+_builddir="_build_awg_${PY_VER}"
+mkdir -pv ${_builddir}
+pushd ${_builddir}
+
+# build this directory
+BUILD_DIR="src/python/awg"
+
+# configure
+cmake \
+	${SRC_DIR} \
+	-DCMAKE_BUILD_TYPE:STRING=Release \
+	-DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
+	-DENABLE_PYTHON${PY_VER%%.*}:BOOL=yes \
+	-DGDS_INCLUDE_DIR=${PREFIX}/include/gds \
+	-DPython3_EXECUTABLE:FILE="${PYTHON}" \
+;
+
+# build
+cmake --build "${BUILD_DIR}" --parallel ${CPU_COUNT} --verbose
+
+# test
+if [[ $build_platform == $target_platform ]]; then
```

Comment:
Tests can also be ran on linux:
```suggestion
if [[ $build_platform == $target_platform || $target_platform == linux-* ]]; then
```

Reply:
This also applies to the other instances of this